game/templates/game/landing.html

```
@@ -104,9 +104,7 @@
       <div class="nav-right">
         {% if user.is_authenticated %}
         <div class="profile-dropdown">
-          <button class="profile-btn">
-              {{ user.username }} ⌄
-          </button>
+          <button class="profile-btn">{{ user.username }} <svg class="profile-arrow" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="3" stroke-linecap="round" stroke-linejoin="round"><path d="M6 9l6 6 6-6"></path></svg></button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)